app/src/main/java/org/example/app/App.java

```diff
@@ -4,7 +4,7 @@
 package org.example.app;
 
 import s2.services.Client;
-import s2.v1alpha.*;
+import s2.v1alpha.S2.*;
```

Comment:
I think we could avoid this if we add a `java_package` directive on the original proto https://github.com/s2-streamstore/s2-protos/blob/main/s2/v1alpha/s2.proto (as was done in the duplicate) -- doesn't seem like a big deal to me to keep like this however

Reply:
oh, we should add that

Reply:
I like the java_package directive idea, personally.